src/zeit/content/article/edit/browser/template.py

```diff
@@ -25,6 +25,17 @@ def render(self):
 </script>""" % self.prefix
         return result
 
+    def reload(self, element=None):
```

Comment:
Das schieb ich mal noch nach zeit.edit...Form in die Basisklasse, wo auch das `def signal` lebt.